### PR TITLE
Change description of ONLY_FULL_GROUP_BY

### PR DESCRIPTION
**Pull request by suyuyi**
Change Description of ONLY_FULL_GROUP_BY

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Change description of ONLY_FULL_GROUP_BY Chinese version, the previous description is wrong
修改 ONLY_FULL_GROUP_BY 部分的描述，原有的描述与实际不一致

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
@tiancaiamao Could you please help review this PR? Thanks!

**Review by tiancaiamao — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

File: sql-mode.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Modes 是用逗号 (',') 间隔开的一系列不同的模式。使用 `SELECT @
 | PIPES_AS_CONCAT | 将 "\|\|" 视为字符串连接操作符（＋）(同CONCAT())，而不视为OR（支持) |
```
Comment (hfxsd) [inline sql-mode.md:24]: L18 中的 SQL 前需要加空格

File: sql-mode.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Modes 是用逗号 (',') 间隔开的一系列不同的模式。使用 `SELECT @
...
 | ANSI_QUOTES | 将 `"` 视为识别符，如果启用 ANSI_QUOTES，只单引号内的会被认为是 String Literals，双引号被解释为识别符，因此不能用双引号来引用字符串（支持）|
 | IGNORE_SPACE | 若开启该模式，系统忽略空格。例如：“user” 和 “user “ 是相同的（支持）|
-| ONLY_FULL_GROUP_BY | 如果 GROUP BY 出现的列并没有在 SELECT，HAVING，ORDER BY 中出现，此 SQL 不合法，因为不在 GROUP BY 中的列被查询展示出来不符合正常现象（支持) |
+| ONLY_FULL_GROUP_BY | 如果未被聚合函数处理或未被 GROUP BY 的列出现在 SELECT，HAVING，ORDER BY 中，此 SQL 不合法，因为不在 GROUP BY 中且未被聚合处理的列被查询展示出来不符合正常现象（支持) |
```
Comment (hfxsd) [inline sql-mode.md:27]:
```suggestion
| ONLY_FULL_GROUP_BY | 如果未被聚合函数处理或未被 GROUP BY 的列出现在 SELECT、HAVING、ORDER BY 中，此 SQL 不合法，因为这种列被查询展示出来不合常规（支持) |
```

File: sql-mode.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ Modes 是用逗号 (',') 间隔开的一系列不同的模式。使用 `SELECT @
...
 | NO_UNSIGNED_SUBTRACTION | 在减运算中，如果某个操作数没有符号，不要将结果标记为UNSIGNED（支持）|
```
Comment (hfxsd) [inline sql-mode.md:28]:
```suggestion
| NO_UNSIGNED_SUBTRACTION | 在减运算中，如果某个操作数没有符号，不要将结果标记为 UNSIGNED（支持）|
```

File: sql-mode.md (hunk 1 above)
Comment (hfxsd) [inline sql-mode.md:24]:
```suggestion
| PIPES_AS_CONCAT | 将 "\|\|" 视为字符串连接操作符（＋）(同 CONCAT())，而不视为 OR（支持) |
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**